pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -1258,3 +1265,13 @@ func runInspect(streams genericiooptions.IOStreams, config *rest.Config, destDir
 	}
 	return nil
 }
+
+func buildPodCommand(
+	volumeCheckerScript string,
+	gatherCommand string,
+) string {
+	// Start the checker in the background,
+	// run the gather command and wait for it to complete/exit.
+	// Make sure all changes are written to disk at the end.
+	return fmt.Sprintf("%s & ( %s ); sync", volumeCheckerScript, gatherCommand)
```

Comment:
I think, we shouldn't have parenthesis. We want to run `gatherCommand` in foreground and once it is completed, we can run `sync`.

Reply:
```suggestion
	return fmt.Sprintf("%s & %s; sync", volumeCheckerScript, gatherCommand)
```

Reply:
That would be better to remove it for now (previous version works like that). If this is not current and we need to update this, this can be achieved in a separate PR.

Reply:
Amended. Had to fix some tests as well.